source/routes/create/bullet.js

```diff
@@ -0,0 +1,58 @@
+const PouchDB = require("pouchdb");
+PouchDB.plugin(require("pouchdb-find"));
+const db = new PouchDB("db");
+
+// Send in json form of the bullet and it gets sent to the database
+module.exports = {
+    "/create/bullet": {
+        methods: ["post"],
+        fn: function(req, res, next) {
+            // Check if every field exists, if not, throw error
+            let requiredFields = ["parentDocId", "user", "signifier", "bulletType", "content", "date"];
+            requiredFields.forEach((jsonField, index) =>{
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7402063)
```suggestion
            requiredFields.forEach((jsonField, index) =>{
```